crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs

```diff
@@ -746,7 +746,12 @@ mod tests {
     fn test_serialize_device_based_strategy() {
         let encryption_settings = all_devices_strategy_settings();
         let serialized = serde_json::to_string(&encryption_settings).unwrap();
-        assert_snapshot!(serialized);
+        with_settings!({ sort_maps => true, prepend_module_to_snapshot => false }, {
```

Comment:
please don't set `sort_maps` here: it's meaningless for when `assert_snapshot` is applied to a string, and even if it were required it should be a different PR.

```suggestion
        with_settings!({prepend_module_to_snapshot => false}, {
```

Reply:
Fixed by https://github.com/matrix-org/matrix-rust-sdk/pull/4625/commits/e8b4b289cc7cc70df7edb79834d82f5678695829